java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -318,8 +342,14 @@ public interface StringBaseCommands {
      * Sets multiple keys to values if the key does not exist. The operation is atomic, and if one or
      * more keys already exist, the entire operation fails.
      *
-     * @apiNote When in cluster mode, all keys in <code>keyValueMap</code> must map to the same hash
-     *     slot.
+     * @apiNote In cluster mode, if keys in <code>keyValueMap</code> map to different hash slots,
```

Comment:
Please add to the second msetnx (line 368)

Reply:
reverted